.github/workflows/staging_queryweaver(staging).yml

```
@@ -0,0 +1,71 @@
+      - uses: actions/checkout@v4
+
+      - name: Set up Python version
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Create and start virtual environment
+        run: |
+          python -m venv venv
+          source venv/bin/activate
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)